tasks/leave_restart_consul.yml

```diff
@@ -0,0 +1,14 @@
+---
+
+- name: Consul leave
+  delegate_to: "{{ item }}"
```

Comment:
```suggestion
  delegate_to: "{{ rolling_restart_host }}"
```

See my above suggestion for context. Also, we may need to add a pipe with `default(omit, true)` for compatibility with ansible-core 2.16.x.

Reply:
could it be the existing `consul_servers` vars instead of a new one?